tests/components/tuya/test_valve.py

```diff
@@ -0,0 +1,61 @@
+"""Test Tuya valve platform."""
```

Comment:
Can we add a couple of tests: `test_turn_on` and `test_turn_off`?

You can look at select platform for a sample:
https://github.com/home-assistant/core/blob/102d6a37c03909cd612983fd7c74a8f318f8b9d1/tests/components/tuya/test_select.py#L63-L72